### ISSUE TITLE
Nothing displaying

### ISSUE THREAD

**Issue by guynwpg** [tomlandia/fuckr#108]
<img width="479" alt="fuckr" src="https://cloud.githubusercontent.com/assets/20740291/23247141/fc2f8b18-f95d-11e6-9c0d-288ac8dc5333.png">

Tried both 64 and 32, same result.

Flushed all appdata,, same result.

**Comment by RobbieTechie** [comment]
Same here, cleared out the AppData directory and started fresh and the same thing shows up.

**Comment by RobbieTechie** [comment]
If you edit the index.html file, where it calls for the google maps script, switch the https to http, that will make it work.  There is an SSL issue it said in the console.
switch:
`<script src="https://maps.googleapis.com/maps/api/js?libraries=places"></script>`
to
`<script src="http://maps.googleapis.com/maps/api/js?libraries=places"></script>`

**Comment by RobbieTechie** [comment]
The nothing displaying is another issue thanks to Windows 10, this thing works just fine on Windows 7 64 bit

**Comment by guynwpg** [comment]
Actually, I'm using Windows 7, 64 bit and got the 'Nothing displaying' issue but using your trick above fixed everything AND let's me switch locations without any problems.

**Comment by Alastor1** [comment]
yeah, can open it.... i do see button, i see messages, but cant see profiles neither see/change my location.... but as Robbie say, Its due to W 10   ._.  feelsbadman, there will be any fix?

**Comment by RobbieTechie** [comment]
I'm not sure why you got that under Windows 7 @guynwpg it only happened to me under Windows 10, as there have been issues with Windows 10 and Chrome for a while now on and off.  I was on Win10 when I saw your post, and tested it.  Removing the s in the https does not really fix it, as when debugging it said there is an SSL issue.  I switched back to Windows 7 and downloaded what was put on fuckr.me and it ran perfect right off the bat.  I did however clear my AppData, as I usually do when testing.

@Alastor1 Tom updated and recoded some things when he uploaded this to fuckr.me.  The version I had been working on does work in Windows 10 though, so I can update it, as I had one issue to fix with offline chat messages that never got uploaded.  When I get that done I will get you the link.  You are on Windows 10 64 bit correct?

**Comment by Alastor1** [comment]
Hi ,yes I am on W10 64 bits :D @RobbieTechie  thank u man.

**Comment by guynwpg** [comment]
@RobbieTechie Hmm well I'll give a breakdown of what I went through and maybe that will shed some light? 

When I saw Tom's build go up, I deleted appdata first and noticed right away the 'nothing display'. Then I went back to your build but strangely, I was getting the same issue, even after deleting appdata again and despite the fact that your build worked perfectly fine before trying Tom's.

Then, after reading your post above, I tried removing 's' from https in the index.html in the program folder belonging to Tom's build, deleted appdata and then Tom's build started working normally (everything was displaying). I noticed there weren't any filters like your build, so tried your build again and again, strangely, yours started working normally again too even though I never edited the index.html file your build's program folder.

I can only conclude that something in the appdata is overriding the configuration in the index.html. That would explain why your build mirrors Tom's build even though I don't touch any files in the program folder belonging to your build. 

I'm assuming Windows 7 and Windows 10 would have unconnected appdata folders so that would also explain why you got different results from both.

Just guessing though as I'm not familiar with programming/coding haha.

**Comment by bootyliciousgeek** [comment]
@RobbieTechie i don't have any problem with your version but tom's won't display any profile. how can i clear appdata but keep the messages? or do you think i should use @RobbieTechie version until tom's version is stable

**Comment by RobbieTechie** [comment]
Tom changed some code, and Windows 10 again didn't agree with that.  As for clearing the AppData and keeping the convos, I'll have to do some testing for that.

**Comment by RobbieTechie** [comment]
@guynwpg I'm not sure.  I know when I first started working with this, to get profiles to show up, you would have to change the views or check and uncheck the online or photo only box, as if something was not functioning properly, but then doing that would get it going.
As for Tom's he changed some code and Windows 10 things there is an SSL error, as far as I'm concerned Windows 10 has been nothing but problems since it came out.  It's recently become a little better but still it seems to be crap for an OS

**Comment by RobbieTechie** [comment]
@Alastor1 
Here is the version I did that works on Windows 10.  I had to fix an issue where offline chat messages were not going through.  I just ran and tested this and it works except the logging out, which still didn't work on Tom's.  If you have issues, just let me know.
https://drive.google.com/file/d/0BxSmknffFGTnMnNDZFk1c3JKTm8/view?usp=sharing

**Comment by Alastor1** [comment]
Oh thank you , im trying it right now, didnt ask me to log in, as soon as I opened it, it logged me in, i can see some guys and can use filters, seems like everything is normal, I cant switch to the account i tried on the android device tho, but im great with this , thank you so much @RobbieTechie   :D

**Comment by grholden** [comment]
Hi guys, not a coder or anything - trying to use the latest .dmg on a Mac and getting this issue that nothing shows up. Thanks for your hard work.

**Comment by bootyliciousgeek** [comment]
the win10 version works fine! @RobbieTechie 
but is it just me or does it require multiple clicks/attempts to open the map when you clicked at the distance? always works fine when you first opened the app but not later

**Comment by RobbieTechie** [comment]
Do you mean if you click the globe on someone's profile @bootyliciousgeek ?  That did seem to happen at times, and at other times it happens right fast.

**Comment by tu156** [comment]
I'm still having the problem with windows 10 not displaying anything. Any suggestions in getting this to work? I  can log in, but see the same info as the picture at the top of this thread. I extract all into a new folder and open the application but get nothing. If I log on on my phone I'm getting messages from people thousands of miles away. I can't change my location on the desktop app though. I type a town, but don't get the normal drop down menu with suggestions. Any ideas?

**Comment by dcminicoder** [comment]
Same here on OS X. Also tried on my windows 10 VM but having the same problems.  Nothing showing up. No map , no profiles.  😅

**Comment by lesterquivers** [comment]
I updated the HTTPS to HTTP and was able to get the screen to show up but nothing else and I cannot enter my zipcode in either. 

Windows 7 - 64

![untitled-1](https://cloud.githubusercontent.com/assets/26077558/23390444/f1c382b0-fd22-11e6-888f-c85f86f0635f.jpg)

**Comment by bootyliciousgeek** [comment]
yeah @RobbieTechie the globe thingy
when you first opened the app, yup one click will open the map
after a while you have to click it multiple times
or sometimes there's no globe icon at all, and the location wasn't refreshed. you have to do some filtering to match the profile's data (e.g. age, or photo/no photo) to get the updated location and the globe icon showing

**Comment by RobbieTechie** [comment]
I have noticed some different behaviors with the app @bootyliciousgeek 
If a profile gets taken off the main grid, it seems the globe will not function, or if the person was on say 58 minutes ago and you have the profile open, then wait until it's over the 1 hour mark, the map won't open their location.  Some people, when they message you, say without a profile pic, and if you've only profiles with pics showing, the profile data won't be read and the distance won't work.  These appear to be some habits from the way grindr now queries profiles.

**Comment by RobbieTechie** [comment]
@lesterquivers Did you previously have a different copy or version of Fuckr on your computer?

**Comment by lesterquivers** [comment]
@RobbieTechie yes I did. Should I delete all the files and re-download? 

do you happen to know if that will erase all the pics I've received?

**Comment by RobbieTechie** [comment]
@lesterquivers  You could delete your AppData\Fuckr folder, which stores the settings, but unfortunately the login, settings, and chat are all stored in the same file, so deleting that would remove old chats.

The other option you have is to edit the file in the fuckr directory called package.json, the first line of text will say name: "Fuckr" change the Fuckr in quotes to something different, like Fuckr2.  That will create a new directory when you run the app again, and you'll run it like it's never been installed.  You'll have to log back in, but you won't be deleting your old chats.  They won't show up, but they will still be on your computer.

**Comment by braadinny** [comment]
@RobbieTechie  - you rock!!. your trick worked.

**Comment by lesterquivers** [comment]
I deleted all the files (entire folder where the app and re-downloaded the app but when I opened it back up it still did not work and all my messages were still there. Am I missing something?

**Comment by RobbieTechie** [comment]
@lesterquivers copy and paste this into your search bar at the bottom of the start menu, then hit enter
c:\users\%username%\AppData\Local\Fuckr
Then highlight and delete the "User Data" folder and delete it.  Now go back and launch Fuckr again and login and it should be good to go.

**Comment by bootyliciousgeek** [comment]
@RobbieTechie i replaced the https line in the 32 bit version but nothing is displaying.

32 bit works for me to log in or log out, but doesn't show any profile and no filter is working.
whereas the 64 bit works in all areas except logging in and out.

could you see if you can somehow marry these two lol
